pkg/internal/testing/controlplane/apiserver.go

```diff
@@ -375,7 +375,11 @@ func (s *APIServer) populateAPIServerCerts() error {
 		return err
 	}
 
-	servingCerts, err := ca.NewServingCert()
+	certNames := []string{"localhost"}
+	if len(s.SecureServing.Address) > 0 {
+		certNames = append(certNames, s.SecureServing.Address)
+	}
+	servingCerts, err := ca.NewServingCert(certNames...)
```

Comment:
Only even when no certnames are passed to `ca.NewServingCert()`, `localhost` is getting added by [default](https://github.com/jkmw/controller-runtime/blob/616b89b5d49212fac50bb31cbdd6903eb4ed65a3/pkg/internal/testing/certs/tinyca.go#L160). Do we modify the condition in `pkg/internal/testing/certs/tinyca.go` instead of here?

Reply:
Would also be possible, of course. Should I implement this or is it a consideration for now?